boilerplate/openshift/golang-osd-operator/standard.mk

```
@@ -57,7 +57,7 @@ GOLANGCI_LINT_CACHE ?= /tmp/golangci-cache
 
 GOLANGCI_OPTIONAL_CONFIG ?=
 
-TESTTARGETS := $(shell ${GOENV} go list -e ./... | egrep -v "/(vendor)/")
+TESTTARGETS ?= $(shell ${GOENV} go list -e ./... | egrep -v "/(vendor)/")
```

It turns out that the CI failure is a result of this variable having changed from "simply expanded" to "recursively expanded" [1]. I don't fully grok it, but I believe it has something to do with the expansion of `${GOENV}` needing to happen only when `TESTTARGETS` is defined, rather than when the `go-generate` target is evaluated. Because evaluating `${GOENV}` triggers a stealth `go mod tidy`?

Anyway, to my understanding, there is no "simply expanded" form of `?=`, so we have to work around it with something like:

```suggestion
ifeq ($(origin TESTTARGETS), undefined)
TESTTARGETS := $(shell ${GOENV} go list -e ./... | egrep -v "/(vendor)/")
endif
```

(See [2] for the equivalence of `?` and this `ifeq` business.)

I tested the above locally and it fixes the issue.

Arguably, we should do a deeper analysis of the relative benefits of simply vs recursively defining `${GOENV}` itself, but that's for another time.

[1] https://www.gnu.org/software/make/manual/html_node/Flavors.html#Flavors
[2] https://www.gnu.org/software/make/manual/html_node/Setting.html#Setting